dt-reports/magic-url-setup.php

```diff
@@ -202,5 +298,47 @@ function is_email_format_valid(email) {
         </script>
         <?php
     }
+
+    private function add_user_app_row( $user_id, $app_key, $app_value, $enabled = false ) {
+        $display = $app_value['settings_display'] ?? true;
+        if ( $display === true ) {
+            $app_user_key = get_user_option( $app_key, $user_id );
+            $app_url_base = trailingslashit( trailingslashit( site_url() ) . $app_value['url_base'] );
+            $app_link = false;
+            if ( $app_user_key ) {
+                $app_link = $app_url_base . $app_user_key;
+            }
+            ?>
+            <div class="user-app-container">
+                <div class="app-name"><?php echo esc_html( $app_value['label'] )?></div>
+                <div class="app-desc"><?php echo esc_html( $app_value['description'] )?></div>
+                <div class="app-toggle" data-url-base="<?php echo esc_url( $app_url_base ) ?>">
+                    <input class="switch-input" id="app_state_<?php echo esc_attr( $app_key )?>" type="checkbox" name="follow_all"
+                           onclick="app_switch('<?php echo esc_attr( $user_id )?>', '<?php echo esc_attr( $app_key )?>');"
+                        <?php echo esc_attr( $enabled ? 'checked' : '' ) ?>
+                    />
+                    <label class="switch-paddle" for="app_state_<?php echo esc_attr( $app_key )?>">
+                        <span class="show-for-sr"><?php esc_html_e( 'Enable', 'disciple_tools' )?></span>
+                        <span class="switch-active" aria-hidden="true" style="color:white;"><?php esc_html_e( 'Yes', 'disciple_tools' )?></span>
+                        <span class="switch-inactive" aria-hidden="false"><?php esc_html_e( 'No', 'disciple_tools' )?></span>
+                    </label>
+                </div>
+                <div class="app-actions" id="app_link_<?php echo esc_attr( $app_key )?>" data-url-base="<?php echo esc_url( $app_url_base ) ?>">
+                    <a class="app-link dt-action-button small button view"
+                       href="<?php echo esc_url( $app_link ) ?>"
+                       title="<?php esc_html_e( 'link', 'disciple_tools' ) ?>">
+                        <i class="fi-link"></i>
+                    </a>
+
+                    <a type="button" class="app-copy dt-tooltip dt-action-button small button copy_to_clipboard"
+                            data-value="<?php echo esc_url( $app_link ) ?>">
+                        <span class="tooltiptext"><?php esc_html_e( 'Copy', 'disciple_tools' ); ?></span>
+                        <img class="dt-icon" alt="copy" src="https://dtplayground.local/wp-content/themes/disciple-tools-theme/dt-assets/images/duplicate.svg">
```

Comment:
Hard coded link. Lets us the same iconv as the Magic links

<img width="402" height="112" alt="Image" src="https://github.com/user-attachments/assets/5396a264-f8dd-4997-96dd-d457c2e5c434" />